lib/auth.ts

```diff
@@ -9,94 +9,94 @@ const { API_URL, AUTH_CALLBACK, CLIENT_ID, CLIENT_SECRET, JWT_KEY, LOGIN_URL } =
 // Used for internal configuration; 3rd party apps may remove
 const apiConfig: ApiConfig = {};
 if (API_URL && LOGIN_URL) {
-    apiConfig.apiUrl = API_URL;
-    apiConfig.loginUrl = LOGIN_URL;
+  apiConfig.apiUrl = API_URL;
+  apiConfig.loginUrl = LOGIN_URL;
```

Comment:
it looks like this file switched to 2 space indenting - I believe we're using `4` for this repo (since at the time that was more common)